docs/nodes/README.md

```diff
@@ -30,6 +30,7 @@ node, and then validate the Avalanche network using an AvalancheGo node.
 | [**Chain Configs**](./maintain/chain-config-flags.md)                            | Chain specific configurations                               |
 | [**Subnet Configs**](./maintain/subnet-configs.md)                               | Subnet specific configurations                              |
 | [**Run C-Chain Offline Pruning**](./maintain/run-offline-pruning.md)             | C-Chain offline pruning                                     |
+| [**Node bootstrap**](./maintain/understand-how-a-node-bootstraps.md)             | Understand how a node bootstraps                                     |
```

Comment:
Not sure this is the right place. This is not a detailed how-to-guide, it's more of an explaination of how bootstrap works, step by step.